docs/Tutorials/EventsAndTriggers.md

```diff
@@ -0,0 +1,80 @@
+\cond NEVER
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+
+# Events and triggers {#tutorial_events_and_triggers}
+
+The "events and triggers" infrastructure (and the related "events and
+dense triggers") provide some control of SpECTRE executable execution
+from the input file contents by running user-specified code on the
+elements.  It does not generally have major effects on the flow of the
+\ref dev_guide_parallelization_core_algorithm "algorithm", but is
+primarily used for observations, such as writing volume data to H5
+files, and minor simulation adjustments.
+
+### Events
+
+An \ref Event "event" is an input-file-creatable object that performs
+some task that (with the exception of the \ref Events::Completion
+"Completion" event), does not directly affect the execution of the
+algorithm on the element.  The effects of events are limited to
+sending messages.  The most commonly used events (such as the \ref
+dg::Events::ObserveErrorNorms "ObserveErrorNorms" event) send data to
+be written to disk, but have no long-term effects on the simulation
+state.  Others, (such as \ref Events::ChangeSlabSize "ChangeSlabSize")
+can have indirect effects when \ref dev_guide_parallelization_actions
+"actions" react to the messages they send.
+
+### Triggers
+
+A \ref Trigger "trigger" is an input-file-creatable object that
+controls when events are run.  They are checked periodically, once per
+Slab in an evolution executable, and once per iteration in an elliptic
+solve.  At each check, a trigger may fire, causing its associated
+events to run, or not.  Triggers must give a consistent result over
```

Comment:
[nitpick] "At each check a trigger may fire, causing its associated events to run or not."